src/screen/ProductPage.tsx

```diff
@@ -114,65 +156,107 @@ export function ProductPage(props: ProductPageProps) {
       <input
         type="button"
         value={inputType === "add" ? "Add" : "Update"}
+        disabled={createProduct.isLoading ? true : false}
```

Comment:
ini bisa langsung gini aja : 

```suggestion
        disabled={createProduct.isLoading}
```

Reply:
updated mas